packages/office-ui-fabric-react/src/components/GroupedList/GroupSpacer.types.ts

```diff
@@ -19,10 +23,16 @@ export interface IGroupSpacerProps {
   indentWidth?: number;
 }
 
+/**
+ * @deprecated unused, to be removed in 7.0. Use {@link IGroupSpacerProps.indentWidth}
```

Comment:
Unsure how to direct folks until I better understand how often folks customize this area if at all.